articles/quickstart/native/android/03-session-handling.md

```diff
@@ -47,21 +47,55 @@ WebAuthProvider.init(auth0)
                 .withScheme("demo")
                 .withAudience(String.format("https://%s/userinfo", getString(R.string.com_auth0_domain)))
                 .withScope("openid offline_access")
-                .start(LoginActivity.this, callback);
+                .start(LoginActivity.this, webCallback);
 ```
 
-## Save the User's Credentials
+## Check for Tokens when the Application Starts
+
+::: panel Learn about refresh tokens
+Before you go further with this tutorial, read the [refresh token documentation](/refresh-token).
+It is important that you remember the following:
+* Refresh tokens must be securely saved.
+* Even though refresh tokens cannot expire, they can be revoked.
+* New tokens will never have a different scope than the scope you requested during the first login.
```

Comment:
New tokens will have the same scope as was originally requested during the first authentication